packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js

```diff
@@ -4,60 +4,45 @@
 
 /* eslint-disable require-jsdoc */
 
-import {SparkPlugin} from '@ciscospark/spark-core';
-import {find} from 'lodash';
+import {StatelessSparkPlugin} from '@ciscospark/spark-core';
 
 import {SIP_URI, MEETINGS} from '../constants';
 
+import MeetingInfoCollection from './collection';
 import MeetingInfoRequest from './request';
 
-const MeetingInfo = SparkPlugin.extend({
-  namespace: MEETINGS,
-  meetingInfoList: {},
-  children: {
-    meetingInfoRequest: MeetingInfoRequest
-  },
+export default class MeetingInfo extends StatelessSparkPlugin {
+  constructor(attrs, options) {
+    super({}, options);
+
+    this.meetingInfoRequest = new MeetingInfoRequest({}, options);
+  }
+
   /**
    * @param {string} id
    * @returns {object}
    * @memberof MeetingInfo
    */
-  getMeetingInfo(id) {
-    if (this.meetingInfoList[id]) {
-      return this.meetingInfoList[id];
-    }
-
-    return find(
-      this.meetingInfoList,
-      (info) => info.sipUrl === id || info.locusId === id || info.userId === id || info.meetingLink === id
-    );
-  },
+  static getMeetingInfo(id) {
+    return MeetingInfoCollection.get(id);
```

Comment:
Instead of making these static methods, I think other files should just import the MeetingInfoCollection directly to operate on it. That would make this more of a util class to only fetch meeting info from the server.